simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/StationStateFlowManager.java

```diff
@@ -0,0 +1,66 @@
+package com.evbox.everon.ocpp.simulator.station;
+
+import com.evbox.everon.ocpp.simulator.station.evse.Connector;
+import com.evbox.everon.ocpp.simulator.station.states.AvailableState;
+import com.evbox.everon.ocpp.simulator.station.states.StationState;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StationStateFlowManager {
+
+    private final Map<Integer, StationState> evsesStates = new HashMap<>();
```

Comment:
Why EVSE states has station state enum? Can we keep all mutating things as part of StationStore?